_posts/07-01-01-Databases.md

```diff
@@ -6,7 +6,7 @@ anchor: databases
 # Databases {#databases_title}
 
 Many times your PHP code will use a database to persist information. You have a few options to connect and interact
-with your database. The recommended option **until PHP 5.1.0** was to use native drivers such as [mysqli], [pgsql],
+with your database. The recommended option **until PHP 5.1.0** was to use native drivers such as [mysql], [pgsql],
```

Comment:
Hmmm, if you change the link there, then you need to change it down on line 17 also.

I suspect there will be varying opinions on whether this change is suitable. Technically you're right: until PHP 5.1, it was `mysql` and not `mysqli`.  But then pointing to the old `mysql` docs is redundant, so the link should still point to `mysqli` despite being named `mysql`. 
For the sake of less ambiguity I'd leave it alone. But that's just my opinion; I'm not the maintainer.

Reply:
You are absolutely right. Thank you for noticing!